.github/workflows/lint-test-sdk.yml

```diff
@@ -28,4 +50,4 @@ jobs:
           distribution: 'adopt'
 
       - name: Run tests
-        run: make test-data && ./gradlew check --no-daemon --stacktrace
+        run: make test-data branchName=${{ env.TEST_DATA_BRANCH_NAME }} && ./gradlew check --no-daemon --stacktrace
```

Comment:
Do we need to adjust our makefile to support this? Would `branchName` get overriden with `main` when it's set in the makefile?

```
## test-data
testDataDir := src/test/resources/shared
tempDir := ${testDataDir}/temp
gitDataDir := ${tempDir}/sdk-test-data
branchName := main
githubRepoLink := https://github.com/Eppo-exp/sdk-test-data.git
.PHONY: test-data
test-data:
	rm -rf $(testDataDir)
	mkdir -p ${tempDir}
	git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} ${gitDataDir}
	cp -r ${gitDataDir}/ufc ${testDataDir}
	rm ${testDataDir}/ufc/bandit-tests/*.dynamic-typing.json
	rm -rf ${tempDir}
```

Reply:
Ok I've learned more about makefiles and passing it in will indeed override 👌